bluetooth_mesh/mesh.py

```diff
@@ -347,3 +352,87 @@ def unpack(cls, app_key: ApplicationKey, net_key: NetworkKey, local_iv_index: in
             transport_msg = AccessMessage.decrypt(app_key, iv_index, ctl, ttl, seq, src, dst, transport_pdu)
         net_message = NetworkMessage(transport_msg)
         return iv_index, seq, net_message
+
+
+mesh_crc = Configuration(
+        width=8,
+        polynomial=0x07,
+        init_value=0xff,
+        final_xor_value=0xff,
+        reverse_input=True,
+        reverse_output=True
+    )
```

Comment:
I think you should be able to create a single `CrcCalculator` instance in GenericProvisioningPDU class:

```
class GenericProvisioningPDU: 
   CRC_CALCULATOR = CrcCalculator(configuration=mesh_crc)
   ...
```
and then `self.CRC_CALCULATOR.calculate_checksum(...)`